public/video-ui/styles/components/_bar.scss

```diff
@@ -62,3 +62,27 @@
   color: white;
   text-align: center;
 }
+
+.publish-bar {
+  font-size: 13px;
+  width: 60%;
```

Comment:
This may fall over in the future. maybe flexbox to allow it to take up the remaining width?